python-package/xgboost/spark/core.py

```diff
@@ -140,6 +142,12 @@
     "base_margin",  # Use pyspark base_margin_col param instead.
 }
 
+# Global prediction names
+Pred = namedtuple(
+    "Pred", ("prediction", "raw_prediction", "probability", "pred_contrib")
+)
+pred = Pred("prediction", "rawPrediction", "probability", "pred_contrib")
```

Comment:
Do we need to keep a consistent naming scheme? `predContrib` v.s. `pred_constrib`, based on the use of `RawPrediction`.

Reply:
Good suggestion. Done